crates/libcontainer/src/rootfs/mount.rs

```diff
@@ -546,29 +546,27 @@ impl Mount {
             PathBuf::from(source)
         };
 
-        if let Err(err) =
-            self.syscall
+        let max_retries: u32 = 3;
+        let mut retries = 0;
+        loop {
+            match self
+                .syscall
                 .mount(Some(&*src), dest, typ, mount_option_config.flags, Some(&*d))
-        {
-            if let SyscallError::Nix(errno) = err {
-                if !matches!(errno, Errno::EINVAL) {
-                    tracing::error!("mount of {:?} failed. {}", m.destination(), errno);
+            {
+                Ok(_) => break,
+                Err(err) => {
+                    if let SyscallError::Nix(errno) = err {
+                        if (matches!(errno, Errno::EINVAL) || matches!(errno, Errno::EBUSY))
+                            && retries < max_retries
+                        {
+                            retries += 1;
+                            std::thread::sleep(std::time::Duration::from_millis(100));
```

Comment:
Is `100ms` reasonable?

Reply:
runc seems to set it to 100ms when doing the same thing.
https://github.com/opencontainers/runc/blob/8d90e3dba696ac787ee64de4445517ddf1063b04/libcontainer/rootfs_linux.go#L1232
But I don't know if this is reasonable 🤔

Reply:
Could you add a comment regarding where `100ms` came from?